python/jupyter_suggestions_core/src/plugins.ts

```diff
@@ -0,0 +1,84 @@
+import {
+  hintIcon,
+  ISuggestionsModel,
+  SuggestionsModel,
+  SuggestionsPanelWidget
+} from '@jupyter-suggestions/base';
+import {
+  ILayoutRestorer,
+  JupyterFrontEnd,
+  JupyterFrontEndPlugin
+} from '@jupyterlab/application';
+import { INotebookTracker } from '@jupyterlab/notebook';
+
+import { ISuggestionsModelToken } from './tokens';
+
+const NAME_SPACE = '@jupyter-suggestions/jupyter-suggestions-core';
+
+export const suggestionsModelPlugin: JupyterFrontEndPlugin<ISuggestionsModel> =
+  {
+    id: `${NAME_SPACE}:suggestion-model`,
+    description: 'The model of the suggestions panel',
+    autoStart: true,
+    requires: [ILayoutRestorer, INotebookTracker],
+    provides: ISuggestionsModelToken,
+    activate: (
+      app: JupyterFrontEnd,
+      restorer: ILayoutRestorer,
+      tracker: INotebookTracker
+    ): ISuggestionsModel => {
+      console.log(`${NAME_SPACE}:suggestion-model is activated`);
+      const model = new SuggestionsModel();
+
+      return model;
+    }
+  };
+
+export const COMMAND_IDS = {
+  /**
+   * Command to add a cell suggestion.
+   */
+  addCellSuggestion: 'jupyter-suggestions:command:add-cell-suggestion'
```

Comment:
Minor nit-pick: the double use of `:` may look unfamiliar as it differs from the pattern used in lab and other extensions